docs/managing-dragonfly/replication.md

```diff
@@ -39,3 +39,88 @@ replicaof no one
 
 This will stop the instance from replicating the primary server but will not discard the dataset it has already replicated. This syntax is useful in cases where the original primary fails. After running `replicaof no one` on a replica of the failed primary, the former replica can be used as the new primary and have its own replicas as a failsafe.
 
+
+## Secure replication with TLS -- Prerequisites
+
+Dragonfly supports replication over TLS. To make it work you need:
+
+1. A valid certificate issued by a CA. We recommend [let's encrypt](https://letsencrypt.org/).
```

Comment:
This is confusing to me. What's the role of the public-CA-signed cert? Which one is it?